src/react-components/misc/useVolumeMeter.js

```diff
@@ -30,7 +30,6 @@ function updateVolume(analyser, meter) {
 export function useVolumeMeter({ analyser, update, updateRate = 50 }) {
   const movingAvgRef = useRef();
   const meterRef = useRef({ levels: [], volume: 0, prevVolume: 0, max: 0 });
```

Comment:
Is this the linter? I think we should be all following the same rules from eslintrc and prettierrc and it doesn't change the format of this file for me... Maybe you have some other additional rules that override those?